### DISCUSSION TITLE
__consumer_offsets Topic in Strimzi Kafka

### DISCUSSION THREAD

**Discussion by teimyBr** [strimzi/strimzi-kafka-operator#9853] [Q&A] [answered]
How to configure __consumer_offsets Topic ? 

Background: 

i want to set a higher retention.ms on this topics that th offset are longer stored in Kafka. 

Is this possible ?

**Comment by scholzj** [comment, marked as answer]
This is Kafka's internal topic. You should probably not change its configuration just like that. You should use the available options (typically `offsets.`) in the Kafka configuration.
